fe/src/main/java/org/apache/doris/qe/SqlModeHelper.java

```diff
@@ -139,21 +143,26 @@ public static String decode(Long sqlMode) throws AnalysisException {
     }
 
     // convert string type SQL MODE to long type that session can store
-    public static Long encode(String sqlMode) throws AnalysisException {
+    public static Long encode(String sqlMode) throws DdlException {
         List<String> names =
                 Splitter.on(',').trimResults().omitEmptyStrings().splitToList(sqlMode);
 
         // empty string parse to 0
         long value = 0L;
         for (String key : names) {
+            if (StringUtils.isNumeric(key)) {
```

Comment:
If there are two combie modes here, it seems that it will pass through this logic.